dashboard/app/views/levels/_external.html.haml

```diff
@@ -0,0 +1,9 @@
+%link{rel: 'stylesheet', href: asset_path('blockly/css/common.css')}
+
+#external
+  :javascript
+    var options = #{@level.properties['options'].to_json}
+
+    $.ajax("#{asset_path @level.properties['href']}").done(function (data) {
+      $('#external').html(data);
+    });
```

Comment:
What's the failure case for this load - does the `#external` element have the helpful "This is taking a long time, you may need to reload the page" message?